script/integration/migrate.rb

```diff
@@ -85,6 +85,44 @@ def migrate_info_checksums(database_url)
   end
 end
 
+def migrate_created_at(database_url)
```

Comment:
There is actually an update on created_at at line 61. Quite hidden actually. Do you think it is better to remove that update?